capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,32 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function getSuggestionsForUserToChooseFrom() {
+  const val = SEARCH_INPUT.value.trim();
+  if (!val) {
+    return;
+  }
+  const fetchStr = '/search?query=' + val;
+  console.log('Fetch String: ', fetchStr);
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    getsAllProposedSuggestions(suggestions, val);
+  });
+}
+
+/**
+ * Sends final college list to the dashboard
+ */
+function sendCollegeInformationToDashboard() {
+  const listItems = document.querySelector('ul').children;
```

Comment:
This feels a bit brittle. What if another `ul` appears in the DOM before this `ul`? Can we refer to the ul by an id here, instead?

Reply:
updated :)